packages/terra-i18n-plugin/package.json

```diff
@@ -28,7 +28,9 @@
     "react-dom": "^15.4.2"
   },
   "dependencies": {
-    "mkdirp": "^0.5.1"
+    "enhanced-resolve": "^3.4.1",
+    "memory-fs": "^0.4.1",
+    "webpack": "^3.6.0"
```

Comment:
these should be development dependencies

Reply:
I think we should update our own usage of webpack in terra-site to webpack v3 before we have plugins using webpack v3. Thoughts @mjhenkes @emilyrohrbough ?